addons/godot_pixelorama_importer/animation_player_import.gd

```diff
@@ -0,0 +1,197 @@
+tool
+extends EditorImportPlugin
+
+const VISIBLE_NAME := "Sprite & AnimationPlayer"
+
+var editor: EditorInterface
+
+
+func _init(editor_interface):
+	editor = editor_interface
+
+
+func get_importer_name():
+	return "com.technohacker.pixelorama.animationplayer"
+
+
+func get_visible_name():
+	return VISIBLE_NAME
+
+
+func get_recognized_extensions():
+	return ["pxo"]
+
+
+# We save directly to stex because ImageTexture doesn't work for some reason
```

Comment:
This comment should probably be removed considering this one works on TSCN files

Reply:
I forgot it, it's removed